L/libbensolve/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libbensolve"
+version = v"2.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://users.fmi.uni-jena.de/~sa67leb/bensolve/files/bensolve-$(version).tgz", "f13c6708974553e346c5426c0471f395fc4de417b55ac9fb07978fa9d268f0a0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bensolve*/
+cc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
+install -Dvm 755 libbensolve.$dlext -t ${libdir}
+install -Dvm 644 *.h -t ${includedir}
+install_license /usr/share/licenses/GPL-3.0+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
Congratulations, you're the first victim of #9393
```suggestion
platforms = supported_platforms(; exclude=p->Sys.isfreebsd(p) && arch(p) == "aarch64")
```